mb_netmgmt/__main__.py

```
@@ -104,6 +104,15 @@ def get_to(self):
             proxy = self.get_proxy(stubs[-1])
             if not proxy:
                 proxy = self.get_proxy(stubs[0])
+            return proxy
+        except (IndexError, AttributeError, requests.RequestException) as e:
+            logging.error(f"Failed to get proxy configuration: {e}")
+            raise
```

what is the point of logging an error when the same exception gets raised anyways? i think it would be cleaner to just let the exception bubble upwards. if it is handled on some upper layer gracefully, we don't care about this message, in case it is unhandled, we get all the relevant information from the stack trace anyways